wordpress-mobile/gutenberg-mobile
# Rename iOS example project to GutenbergDemo

`gutenberg` side PR: https://github.com/WordPress/gutenberg/pull/25702

This PR implements the changes from https://github.com/WordPress/gutenberg/pull/25702

**Note:** You might have a lone `gutenberg/packages/react-native-editor/ios/gutenberg.xcworkspace` folder after checkout. Remove this or `npm run core ios` will fail.

**To test:**
- Checkout
- `rm -rf `gutenberg/packages/react-native-editor/ios/Pods   gutenberg/packages/react-native-editor/ios/build`.
- Open `gutenberg/packages/react-native-editor/ios/GutenbergDemo.xcworkspace`.
- `Clean build folder` from Xcode.
- `npm install`
- `npm run core ios`.
  - Check that the project build and run on the simulator as expected.
- Build and Run from Xcode.
  - Check that it works as expected.
- Modify a .swift file on GutenbergDemo project.
- Build and Run again.
  - Check that everything works as expected.

PR submission checklist:
